packages/frontend-shared/src/gql-components/HeaderBarContent.vue

```diff
@@ -271,7 +268,7 @@ fragment HeaderBar_HeaderBarContent on Query {
     branch
     isLoadingNodeEvents
   }
-  projectRootFromCI
+  isGlobalMode,
```

Comment:
Doesn't make a difference, since the character is ignored:

```suggestion
  isGlobalMode
```